fsspec/tests/test_fuse.py

```diff
@@ -5,37 +5,59 @@
 from fsspec.fuse import run
 import time
 
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
The logger is only for CI debugging purposes?

Reply:
Yes